src/wr_led_matrix/README.md

```diff
@@ -0,0 +1,15 @@
+# Changes
```

Comment:
This file was just instructions for making the package, it doesn't really need to be in the final project.

Reply:
This file is removed.